kingpin/actors/aws/s3.py

```diff
@@ -378,6 +378,46 @@ class LifecycleConfig(SchemaCompareBase):
     }
 
 
+class NotificationConfiguration(SchemaCompareBase):
+
+    """Provides JSON-Schema based validation of the supplied Notification Config.
+
+   .. code-block:: json
+
+      {
+         "queueconfigurations": [
+             {
+                "queue_arn": "ARN of the SQS queue",
+                "events": ["s3:ObjectCreated:*"],
+             }
+        ]
+      }
+   """
+
+    SCHEMA = {
+        'type': ['object', 'null'],
+        'required': ['queueconfigurations'],
+        'properties': {
+            'queueconfigurations': {
+                'type': ['array'],
+                'items': {
+                    'type': 'object',
+                    'additionalProperties': False,
+                    'required': ['queuearn', 'events'],
+                    'properties': {
+                        'queue_arn': {
+                            'type': 'string'
+                        },
+                        'events': {
```

Comment:
Can you more fully define this? It should be defined out as an array of strings..

Reply:
done